tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -251,6 +251,8 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 					return false
 				}
 
+				Expect(utils.PipelineRunFailed(releasePr)).NotTo(BeTrue(), fmt.Sprintf("did not expect PipelineRun %s:%s to fail", releasePr.GetNamespace(), releasePr.GetName()))
+				Expect(utils.PipelineRunFailed(additionalReleasePr)).NotTo(BeTrue(), fmt.Sprintf("did not expect PipelineRun %s:%s to fail", additionalReleasePr.GetNamespace(), additionalReleasePr.GetName()))
 				return releasePr.HasStarted() && releasePr.IsDone() && releasePr.Status.GetCondition(apis.ConditionSucceeded).IsTrue() &&
 					additionalReleasePr.HasStarted() && additionalReleasePr.IsDone() && additionalReleasePr.Status.GetCondition(apis.ConditionSucceeded).IsTrue()
 			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
```

Comment:
@kasemAlem to fully address RHTAPBUGS-121 (which I think this PR should be linked to), I think this fix needs to applied to all release test specs that are polling release pipelinerun. Addressing this in "pyxis" test is not enough imo

Reply:
@psturc , 
I made a change on all release tests in release pipelineruns , 
WDYT?